ts/session/apis/open_group_api/sogsv3/sogsV3RoomInfos.ts

```diff
@@ -10,7 +10,7 @@ import { DURATION } from '../../../constants';
 
 export const getAllRoomInfos = async (roomInfos: OpenGroupV2Room) => {
   const result = await OnionSending.sendJsonViaOnionV4ToSogs({
-    blinded: true,
+    blinded: false,
```

Comment:
I thought we only support blinded communities now?

Reply:
we dont, and won't for some time at least (we need blinded 25)